src/formatDate.js

```diff
@@ -8,7 +8,24 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldSeparator = Array.from(fromFormat).pop();
```

Comment:
Using `Array.from(fromFormat)` is unnecessary here, as `fromFormat` is already an array. You can directly access the last element using `fromFormat[fromFormat.length - 1]`.